addons/web/static/lib/owl/owl.js

```diff
@@ -2818,8 +2818,10 @@
         set(mode) {
             QWeb.dev = mode === "dev";
             if (QWeb.dev) {
-                const url = `https://github.com/odoo/owl/blob/master/doc/reference/config.md#mode`;
-                console.warn(`Owl is running in 'dev' mode.  This is not suitable for production use. See ${url} for more information.`);

```

Comment:
this isn't the right place to do this change (will be lost the next time we update owl)

Reply:
@xmo-odoo seems like you r+ the PR without addressing this point on the owl repo

Reply:
@sdegueldre odoo/owl#880